File: CS50/sentimental-hello/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int


h = get_int("Height: ")


while h < 1 or h > 8:
    h = int(input("Height: "))


for i in range(h):
    for _ in range (h-i-1):
        print(" ", end="")
    for _ in range (i+1):
        print ("#", end="")
    print()

File: CS50/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    //Get Size of the piramid
    int n;
    do
    {
        n = get_int ("Choose the Height between 1 and 8: ");
    }

    //Print the piramid
    while (n < 1 || n > 8);

       for (int i = 1; i <= n; i++)

       {
        
        for (int j = 1; j <= n - i; j++)
        {
            printf(" ");
        }

        for (int j = 1; j <= i; j++)
        {
            printf("#");
        }
        printf("\n");
    }

}